stdlib/sql/to.go

```diff
@@ -207,38 +222,95 @@ func (t *ToSQLTransformation) Finish(id execute.DatasetID, err error) {
 	t.d.Finish(err)
 }
 
+type translationFunc func(f flux.ColType, colname string) (string, error)
+
+func correctBatchSize(batchSize, numberCols int) int {
+	/*
+		BatchSize for the DB is the number of parameters that can be queued within each call to Exec.
+		As each row you send has a parameter count equal to the number of columns (i.e. the number of "?" used in the insert statement), and some DBs
+		have a default limit on the number of parameters which can be queued before calling Exec; SQLite, for example, has a default of 999 (can only be changed
+		at compile time).
+
+		So if the row width is 10 columns, the maximum Batchsize would be:
+		(999 - row_width) / row_width = 89 rows.
+
+		Sending more would result in the call to Exec returning a "too many SQL variables" error, and the transaction would be rolled-back / aborted
+	*/
+
+	if batchSize < numberCols {
+		return numberCols
+	}
+	return (batchSize - numberCols) / numberCols
```

Comment:
I could be wrong if I missed something to understand this 999 cap thing right,  but I think this should be
```suggestion
	return batchSize / numberCols
```

Reply:
No - you need to take account of the initial column width - the column names used as part of the insert statement - if you do it this way, you exceed the limit. At least, that is what I found during testing.

Reply:
Do you mean the column names are also in this 999 allowed variables?
If I have a SQL says `INSERT INTO T (col) VALUES (1)` it actually takes 2 slots (1 column name + 1 column value)?

Reply:
Yes exactly - that initial slot you used for col, would need to be taken into account